### ISSUE TITLE
fps_max 999

### ISSUE THREAD

**Issue by devFrostie** [Ochii/cso2-launcher#58]
CS:O2 fps limit value is 240, but CS games are not.

can you make fps limit value to unlimited?

**Comment by dounai2333** [comment]
the same way we use -masterip to start a game, you can add +fps_max 0 (or 999, whatever one) to your startup.

**Comment by dounai2333** [comment]
the same way we use -masterip to start a game, you can add +fps_max 0 (or 999, whatever one) to your startup.

**Comment by devFrostie** [comment]
> the same way we use -masterip to start a game, you can add +fps_max 0 (or 999, whatever one) to your startup.

Yes, but Ingame doesn't apply. lobby can set 999 but value will be replaced when join the game.

**Comment by dounai2333** [comment]
> > the same way we use -masterip to start a game, you can add +fps_max 0 (or 999, whatever one) to your startup.
> 
> Yes, but Ingame doesn't apply. lobby can set 999 but value will be replaced when join the game.

but accord to my test, as long as you don't change the settings after change fps_max, everything will be fine, and it can even be saved to your next start the game
so what's wrong? (as long as you click ok after opening the option, that means you have changed the settings. if you click cancel or otherwise avoid clicking ok, it will not change)

**Comment by Vad1mCSO2** [comment]
Adjust can edit jpex FFD
![image](https://user-images.githubusercontent.com/76803825/111867722-72e42f00-89a8-11eb-837d-618b7281ac40.png)
![image](https://user-images.githubusercontent.com/76803825/111867737-81cae180-89a8-11eb-8053-78a3a657c594.png)

**Comment by Vad1mCSO2** [comment]
CSO2\Data\cstrike\ui_gfx\addonui\addonpopup.gfx
scripts\SystemOptionAppGraphicView

**Comment by BKVad1m** [comment]
EU/RU Updated July 3rd 2021
